### ISSUE TITLE
cache entry not created in cf_staticfilecache table

### ISSUE THREAD

**Issue by Micha-sus** [lochmueller/staticfilecache#294]
## Bug Report

**Current Behavior**
Static file for page is created in typo3temp/ folder, but corresponding database entry in cf_staticfilecache table is missing. This results in staticfilecache not removing the cached page from typo3temp/ when clearing the page cache via backend or CacheManager API and the outdated page will get served to the client.

**Expected behavior/output**
Clearing the page cache always removes the corresponding files in typo3temp/.

**Environment**
- TYPO3 version: 9.5.18
- staticfilecache version: 11.1.6
- TYPO3 runs in composer mode
- PHP Version: 7.3.17
- MySQL Version: 5.7.30

**Additional context**
This bug only occured once since staticfilecache was installed and only for one page. On other pages the caching behavior was as expected.

**Related Issues**
#112

**Comment by lochmueller** [comment]
Hey @Micha-sus 
do you use typo3_console for deployment? This extension deleted all 'cf_*' tables, without calling the Caching Framework (directly). That is the reason, why you have a option in the extension configuration, to change the tablenames to "sfc_*" for StaticFileCache. Otherwise I saw no reason, why files are generated and DB entry is missing.
Regards,
Tim

**Comment by Micha-sus** [comment]
Hi @lochmueller,
thanks for your quick reply. We don't use typo3_console or any other third party extension that could clear the cf_* tables without using the caching framework. Also all other entries in the cf_staticfilecache table were still present, only one entry was missing, so I think there was a problem with creating the entry. I checked the error log but there was no DB error or any other hint.

To be safe, I changed the table prefix to "sfc_", thanks for the tip! If the error occurs again, I'll update this issue.

Kind regards
Micha
